content/src/content/jcr_root/apps/core/wcm/components/pdfviewer/v1/pdfviewer/README.md

```diff
@@ -0,0 +1,67 @@
+<!--
+Copyright 2020 Adobe
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+PDF Viewer (v1)
+====
+PDF Viewer for Adobe Document Cloud component written in HTL.
+
+## Features
+
+* Allows embedding PDF files using the View SDK
+* Configurable features for controlling appearance and functionality. 
+
+### Use Object
+The PDF Viewer component uses the `com.adobe.cq.wcm.core.components.models.PdfViewer` Sling model as its Use-object.
+
+### Context Aware Config
+The PDF Viewer component uses the context aware config `com.adobe.cq.wcm.core.components.internal.services.pdfviewer.PdfViewerCaConfig` with the following properties:
+
+1. `clientId` - **Required** - the client key obtained by registering for the view sdk api - [https://www.adobe.com/go/dcsdks_credentials](https://www.adobe.com/go/dcsdks_credentials)
+2. `reportSuiteId` - Adobe Analytics ID
+
+### Edit Dialog Properties
+The following properties are written to JCR for this Document Cloud Viewer component and are expected to be available as `Resource` properties:
```

Comment:
should be renamed:
Document Cloud Viewer component -> PDF Viewer component

Reply:
Done in a691b53